cmd/service.go

```diff
@@ -102,7 +102,12 @@ func (m *ServerMux) handleServiceSignals() error {
 			}
 			runExitFn(nil)
 		case serviceStop:
-			log.Println("Gracefully stopping... (press Ctrl+C again to force)")
+			// Extra space will prevent showing "^CExiting" and rather show "^C Exiting" instead.
+			log.Println(" Exiting")
```

Comment:
The `^C` is not the only way to get to get to this line in the code. A SIGTERM may also be sent to cause this line to execute. In that case, printing with a leading space is weird - also, there is no info on why it is exiting.